src/3box.js

```diff
@@ -167,6 +208,28 @@ class Box extends BoxApi {
     return this.spaces[name]
   }
 
+  /**
+   * Join a thread. Use this to start receiving updates
+   *
+   * @param     {String}    space                   The name of the space for this thread
+   * @param     {String}    name                    The name of the thread
+   * @param     {Object}    opts                    Optional parameters
+   * @param     {String}    opts.firstModerator     DID of first moderator of a thread, by default, user is first moderator
+   * @param     {Boolean}   opts.members            join a members only thread, which only members can post in, defaults to open thread
+   * @param     {Boolean}   opts.noAutoSub          Disable auto subscription to the thread when posting to it (default false)
+   * @param     {Boolean}   opts.ghost              Enable ephemeral messaging via Ghost Thread
+   * @param     {Number}    opts.ghostBacklogLimit  The number of posts to maintain in the ghost backlog
+   * @param     {Array<Function>} opts.ghostFilters Array of functions for filtering messages
+   *
+   * @return    {Thread}                  An instance of the thread class for the joined thread
+   */
+  async joinThread (space, name, opts) {
```

Comment:
seems confusing to have this func here with the same name, considered have openThread and joinThread or still in a space like space.openThread? but i guess if in space then space with need an open/auth flow, so maybe here but  openThread

Reply:
So just rename this fn to `openThread`? The thing that might become confusing is to have two different names for two methods that essentially does the same. Note that if you join a thread from here and then call `box.auth` for the given space you will automatically be able to post in this thread.

Reply:
On second thought, `openThread` here makes alot of sense. Going to change it!